zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -0,0 +1,1010 @@
+                read_result = read_metadata(metadataPath, lv_size)
+            if read_result.is_usable():
+                payload = read_result.payload.decode('utf-8') \
+                    if isinstance(read_result.payload, bytes) else read_result.payload
+                logger.debug("read vm metadata from %s (%d bytes)",
+                             metadataPath, len(payload) if payload else 0)
+                return {'metadata': payload}
+            else:
+                logger.warn("metadata LV %s is not usable: status=%s, error=%s",
+                            metadataPath, read_result.status, read_result.error)
+                return {'metadata': None}
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)